### ISSUE TITLE
GTK-based application using BT protocol product selection issue

### ISSUE THREAD

**Issue by sergey-gra** [Jutta-Proto/protocol-bt-cpp#8]
When a GTK-based program employs the Bluetooth (BT) protocol, the program functions correctly during the selection of the first product. However, when the selection is attempted for the second time, the program repeats the selection of the first product.

**Comment by sergey-gra** [comment]
Upon conducting a thorough investigation of the source code, I discovered that the function 'request_coffee' in the 'CoffeeMaker.cpp' file is associated with the static constant vector of unsigned 8-bit integers named 'command', which is generated via the function 'from_hex_string(commandHexStr)'. I observed that during the second execution of the 'request_coffee' function, the 'from_hex_string' function was not invoked. However, upon removing the 'static' keyword from the declaration of the 'command' vector, the issue was resolved and the function executed successfully.
